libs/s25main/GamePlayer.cpp

```diff
@@ -74,10 +74,7 @@ GamePlayer::GamePlayer(unsigned playerId, const PlayerInfo& playerInfo, GameWorl
     global_inventory.clear();
 
     // Statistiken mit 0en füllen
-    memset(&statistic[StatisticTime::T15Minutes], 0, sizeof(statistic[StatisticTime::T15Minutes]));
-    memset(&statistic[StatisticTime::T1Hour], 0, sizeof(statistic[StatisticTime::T1Hour]));
-    memset(&statistic[StatisticTime::T4Hours], 0, sizeof(statistic[StatisticTime::T4Hours]));
-    memset(&statistic[StatisticTime::T16Hours], 0, sizeof(statistic[StatisticTime::T16Hours]));
+    memset(&statistic, 0, sizeof(statistic));
```

Comment:
can we use some methods to clear the statistics somehow instead of memset?

Reply:
It is an array of arrays of PODs, so memset is fine here. Can't add a method there, since it's an array. We could use `statistic = {}` which has the same effect, but IMO memset is more clear.

Reply:
`statistic = {}` would be less error prone